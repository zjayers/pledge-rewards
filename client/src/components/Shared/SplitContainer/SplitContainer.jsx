import React from 'react';
import SplitContainer from './SplitContainer.style';

export default ({ children }) => {
  return <SplitContainer>{children}</SplitContainer>;
};
