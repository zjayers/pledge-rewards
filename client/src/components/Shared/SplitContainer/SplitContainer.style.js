import styled from 'styled-components';

export default styled.div`
  width: 100%;
  display: grid;
  grid-template-columns: 1fr 1fr;
`;
