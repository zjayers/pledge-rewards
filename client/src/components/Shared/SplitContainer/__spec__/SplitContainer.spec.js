import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import SplitContainer from '../SplitContainer';

describe('SplitContainer Component', () => {
  let component;

  beforeEach(() => {
    component = mount(<SplitContainer />);
  });

  afterEach(() => {
    component.unmount();
  });

  it('should render without failure', () => {
    expect(component).toBeDefined();
  });

  it('should match the test snapshot', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
