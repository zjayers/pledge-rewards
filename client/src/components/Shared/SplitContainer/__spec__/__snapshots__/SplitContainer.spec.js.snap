// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SplitContainer Component should match the test snapshot 1`] = `
<_default>
  <SplitContainerstyle___default>
    <div
      className="SplitContainerstyle___default-sc-1ep4whv-0 hdmVpp"
    />
  </SplitContainerstyle___default>
</_default>
`;
